lang/python/python3/patches/008-distutils-use-python-sysroot.patch

```diff
@@ -0,0 +1,14 @@
+--- a/Lib/distutils/command/build_ext.py
++++ b/Lib/distutils/command/build_ext.py
+@@ -233,7 +233,10 @@ class build_ext(Command):
+         if (sysconfig.get_config_var('Py_ENABLE_SHARED')):
+             if not sysconfig.python_build:
+                 # building third party extensions
+-                self.library_dirs.append(sysconfig.get_config_var('LIBDIR'))
++                libdir = sysconfig.get_config_var('LIBDIR')
++                if "_python_sysroot" in os.environ:
++                    libdir = os.environ.get("_python_sysroot") + libdir
```

Comment:
maybe use `os.path.join()`
it's a little more robust;
something like:
```
python_sysroot = os.environ.get("_python_sysroot", None)
if (python_sysroot):
   libdir = os.path.join(python_sysroot, libdir)
```

Reply:
I am wondering, if we could adjust`libdir = sysconfig.get_config_var('LIBDIR')` somehow to include `_python_sysroot`;
this looks like a good fix for now, but we may get something similar later

Reply:
ah
nevermind;
saw your comment here https://github.com/openwrt/packages/issues/9275#issuecomment-504563916

this looks fine;
maybe we'll try to upstream this at some point;